sys-apps/firejail/firejail-0.9.64.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6..9} )
+
+inherit toolchain-funcs python-single-r1
+
+if [[ ${PV} != 9999 ]]; then
+	KEYWORDS="~amd64 ~x86"
+	SRC_URI="https://github.com/netblue30/${PN}/releases/download/${PV}/${P}.tar.xz"
+else
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/netblue30/firejail.git"
+	EGIT_BRANCH="master"
+fi
+
+DESCRIPTION="Security sandbox for any type of processes"
+HOMEPAGE="https://firejail.wordpress.com/"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="apparmor +chroot contrib +dbusproxy +file-transfer +globalcfg +network +overlayfs +private-home +suid test +userns +whitelist x11"
+RESTRICT="!test? ( test )"
+
+RDEPEND="apparmor? ( sys-libs/libapparmor )
+	contrib? ( ${PYTHON_DEPS} )
+	dbusproxy? ( sys-apps/xdg-dbus-proxy )"
+
+DEPEND="${RDEPEND}
+	!sys-apps/firejail-lts
```

Comment:
FYI this blocker needs to be in RDEPEND or it won't work properly.

Reply:
> FYI this blocker needs to be in RDEPEND or it won't work properly.

Huh, OK, it seemed to behave right (I can't install firejail-lts while firejail is installed, and vice versa) but I will take your word for it.

firejail-lts specifies the reciprocal block the same way as this, but it hasn't had a release so I haven't made any changes to it. If it's necessary, I can add an -r bump just to fix its RDEPEND; if not, I will just get that next time there is a new -lts.

Reply:
I'm pretty sure I hit this issue and that's why I noticed it.
https://devmanual.gentoo.org/general-concepts/dependencies/ check "Blockers" and especially the big red warning note in there.

Reply:
Ah, gotcha, thanks!